### ISSUE TITLE
PLEASE does ANYONE have a freeplay script using pycozmo.

### ISSUE THREAD

**Issue by capy-dev** [zayfod/pycozmo#66]
I want to keep my cozmo on 24/7, it would be AWESOME, I cannot find out how to do it using pycozmo, and I can't install ADB on a  raspberry pi, thank you all for the help.

**Comment by bp88dev** [comment]
i'm making one rn

**Comment by TycoonCoder** [comment]
How is this going?

**Comment by bp88dev** [comment]
I have the script if you want it, it's done but super unoptimized

**Comment by TycoonCoder** [comment]
That’d be great thank you

**Comment by bp88dev** [comment]
For some reason it has like a 50/50 chance of when you run it cliff detection doesn't work so to prevent your cozmo from falling to the floor, pick him up and keep rerunning the script if he does nothing. If there's any errors that pop up with the script, post an issue, otherwise the code will be uploaded to [here](https://github.com/bp88dev/deskmo) whenever PyCharm decides to finish whatever it's doing

<img width="673" height="178" alt="Image" src="https://github.com/user-attachments/assets/8be1fce3-ce7f-4b70-a769-b5e8cf24362f" />

**Comment by bp88dev** [comment]
nevermind it'll be a minute

<img width="449" height="132" alt="Image" src="https://github.com/user-attachments/assets/b10249bd-4080-437b-bd20-092be5cff786" />

**Comment by bp88dev** [comment]
Alright, the code is uploaded! Enjoy!

**Comment by TycoonCoder** [comment]
Thanks so much! I'm a little confused on how to use the project. Do you have any clarification on that? Thanks!

**Comment by bp88dev** [comment]
In theory, you should just be able to run main.py in the project's root folder, if you need a requirements.txt to install I can probably make one

Sent from [Proton Mail](https://proton.me/mail/home) for Android.

-------- Original Message --------

**Comment by bp88dev** [comment]
Sorry for not responding to your issue on the repo, I was assuming it'd send me an email but apparently not. I just downloaded the mobile app so hopefully that works better.

**Comment by TycoonCoder** [comment]
You are fine! So I just need to clone the entire repo? Also a requirements file would be a big help, thank you very much.

**Comment by bp88dev** [comment]
Yes, just clone the repo, and I just put a requirements.txt file in there that you can install with `python3 -m pip install -r requirements.txt`

**Comment by TycoonCoder** [comment]
Thanks so much!Sent from my Apple WatchOn Dec 22, 2025, at 6:25 PM, Benji ***@***.***> wrote:bp88dev left a comment (zayfod/pycozmo#66)
Yes, just clone the repo, and I just put a requirements.txt file in there that you can install with python3 -m pip install -r requirements.txt
